updater/updater.go

```diff
@@ -168,17 +177,32 @@ func run() error {
 		log.Info("finished deploy of system app")
 	}
 
-	// deploy all other apps
+	// deploy all other apps (including provisioned Redis apps)
 	apps, err := client.AppList()
 	if err != nil {
 		log.Error("error getting apps", "err", err)
 		return err
 	}
 	for _, app := range apps {
+		log := log.New("name", app.Name)
+
+		if strings.HasPrefix(app.Name, "redis-") {
```

Comment:
These apps are actually [system apps](https://github.com/flynn/flynn/blob/master/appliance/redis/cmd/flynn-redis-api/main.go#L209-L213) so do you think checking system + `redis-` prefix is enough?

We should probably have more explicit metadata (e.g. `app.Meta["flynn-redis"] = true`) but we will still need to support the legacy apps without the metadata (or just accept we will not in-place update them).